src/hotspot/share/gc/serial/defNewGeneration.hpp

```diff
@@ -155,6 +155,12 @@ class DefNewGeneration: public Generation {
     return n > alignment ? align_down(n, alignment) : alignment;
   }
 
+  // If "p" is in the space, returns the address of the start of the
+  // "block" that contains "p".  We say "block" instead of "object" since
+  // some heaps may not pack objects densely; a chunk may either be an
+  // object or a non-object.  If "p" is not in the space, return null.
+  HeapWord* block_start_const(const ContiguousSpace* cs, const void* p) const;
```

Comment:
Can this be static function in cpp? (It'd be good if it is not exposed, even as private api.)

Reply:
> Can this be static function in cpp? 

I marked it as static just now.

> (It'd be good if it is not exposed, even as private api.)

It is in the default (private) segment now.

Reply:
My prev msg was probably unclear: I meant instead of adding `block_start_const` to the class, defining them as static function in cpp. For example, `fill_range` in `serialBlockOffsetTable.cpp`.

If possible, the (private) api will be smaller.